apps/deploy-web/src/components/deployments/DeploymentDepositModal.tsx

```diff
@@ -167,8 +168,8 @@ export const DeploymentDepositModal: React.FunctionComponent<Props> = ({ handleC
     setError("");
     clearErrors();
     const deposit = denomToUdenom(amount);
-    const uaktBalance = walletBalances?.uakt || 0;
-    const usdcBalance = walletBalances?.usdc || 0;
+    const uaktBalance = walletBalance?.balanceUAKT || 0;
```

Comment:
should we fall back to default in the hook?

Reply:
This shouldn't happen. I added a check to disable the button to submit, but I prefer to return null for the wallet balance object as we check in some places to render stuff or not.